_posts/2021-12-27-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: post
+title:  "Trino 2021 Wrapped: A Year of Growth"
```

Comment:
Trino wraps 2021 up as a year of growth

Reply:
I'm making a call out to a recent trend in applications to summarize the year started by Spotify which they call Wrapped.

https://www.spotify.com/us/wrapped/